src/ResourceManager/Compute/Commands.Compute/Strategies/Network/LoadBalancerStrategy.cs

```diff
@@ -0,0 +1,293 @@
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.Common.Strategies.Network
+{
+
+    public static class LoadBalancerStrategy
+    {
+        public const string Dynamic = "Dynamic";
+        public const string Static = "Static";
+        public const string LoadBalancerRuleName = "loadBalancingRules";
+        public const string LoadBalancerChildResourceId = "/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.Network/loadBalancers/{2}/{3}/{4}";
+        public const string LoadBalancerInBoundNatRuleName = "inboundNatRules";
+        public const string LoadBalancerInboundNatPoolName = "inboundNatPools";
+        public const string LoadBalancerFrontendIpConfigName = "frontendIPConfigurations";
+        public const string LoadBalancerProbeName = "probes";
+        public const string LoadBalancerBackendAddressPoolName = "backendAddressPools";
+        public const string ResourceGroupNotSet = "ResourceGroupNotSet";
+        public const string LoadBalancerNameNotSet = "LoadBalancerNameNotSet";
+
+        public static ResourceStrategy<LoadBalancer> Strategy { get; }
+            = NetworkStrategy.Create(
+                "load balancer",
+                "loadBalancers",
+                client => client.LoadBalancers,
+                (o, p) => o.GetAsync(
+                    p.ResourceGroupName, p.Name, null, p.CancellationToken),
+                (o, p) => o.CreateOrUpdateAsync(
+                    p.ResourceGroupName, p.Name, p.Model, p.CancellationToken),
+                _ => 30);
+
+        public static ResourceConfig<LoadBalancer> CreateLoadBalancerConfig(
+            this ResourceConfig<ResourceGroup> resourceGroup,
+            string name,
+            string froontendPoolName,
+            string backendPoolName,
+            IList<string> zones,
+            ResourceConfig<PublicIPAddress> publicIPAddress,
+            NestedResourceConfig<Subnet, VirtualNetwork> subnet)
+                => Strategy.CreateResourceConfig(
+                    resourceGroup: resourceGroup,
+                    name: name,
+                    createModel: subscriptionId =>
+                    {
+                        var frontEndConfig = CreateFrontendIpConfig(
+                            froontendPoolName,
+                            subscriptionId,
+                            subnet.GetId(subscriptionId).IdToString(),
+                            publicIPAddress.GetId(subscriptionId).IdToString(),
+                            privateIpAddress: null,
+                            zones: zones);
+
+                        var backendConfig = CreateBackendAddressPoolConfig(backendPoolName, subscriptionId);
+
+                        var lb = new LoadBalancer();
+
+                        lb.FrontendIPConfigurations = new List<FrontendIPConfiguration>();
+                        lb.FrontendIPConfigurations.Add(frontEndConfig);
+
+                        lb.BackendAddressPools = new List<BackendAddressPool>();
```

Comment:
BackendIPConfigurations is a nested resource. We can fix it later.

Reply:
will fix now